openedx/features/learner_profile/tests/views/test_learner_profile.py

```diff
@@ -120,6 +121,19 @@ def test_redirect_view(self):
         with override_settings(PROFILE_MICROFRONTEND_URL=profile_url):
             with override_waffle_flag(REDIRECT_TO_PROFILE_MICROFRONTEND, active=True):
                 profile_path = reverse('learner_profile', kwargs={'username': self.USERNAME})
+
+                # Test with waffle flag active, site setting disabled
+                response = self.client.get(path=profile_path)
+                for attribute in self.CONTEXT_DATA:
+                    self.assertIn(attribute, response.content)
+
+                # Test with waffle flag active, site setting enabled
+                site_domain = 'othersite.example.com'
+                self.set_up_site(site_domain, {
+                    'SITE_NAME': site_domain,
+                    'ENABLE_PROFILE_MICROFRONTEND': True
+                })
+                self.client.login(username=self.USERNAME, password=self.PASSWORD)
```

Comment:
Why did this change require logging in the user here?

Reply:
I think it is a product of how the Django test client manages authn and sites.